prebid-server/pbs-modules/greenbids-real-time-data.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: page_v2
+page_type: pbs-module
+title: Prebid Server Greenbids Real Time Data Module
+display_name : Greenbids Real Time Data Module
+sidebarType : 5
+---
+
+# Greenbids Real Time Data Module
+
+## Overview
+
+Greenbids Real Time Data module filters bidders 
+SSPs listed in the `imp[].ext.prebid.bidder`
+of the bid request. To perform the filtering the module uses the ML pipeline that outputs the probability 
+of bid per SSP for each `imp` for the given bid request. 
+Then this probability of bid is compared with the threshold to ensure the necessary level of filtering for a partner publisher.
```

Comment:
I don't know if there is some linting rules for markdown enforced of this repository but here you have an inconsistent line wrapping.
As a personal preference, I break the paragraphs on phrases, it makes more sense during review (the line is changed if you changed the phrase, not only if you added a word two phrase earlier which require to reformat your whole paragraph).